NuRadioMC/EvtGen/NuRadioProposal.py

```diff
@@ -269,7 +254,7 @@ def shower_properties(particle):
 
     return shower_type, code, name
 
-def GetSecondaries(Elepton, lepton_code, random_seed=None):
+def GetSecondaries(Elepton, lepton_code, random_seed=None, config_file='config_PROPOSAL.json'):
 
```

Comment:
Someone probably should have caught this earlier. Please only use camel case for Classes and get_secondaries() for functions. This is python standard, I believe.

Reply:
Changed. Also the name of the class secondary_interactions is now SecondaryInteractions.